app.py

```diff
@@ -1313,7 +1313,8 @@ def index(level, program_id):
 
     customizations = {}
     if current_user()['username']:
-        customizations = DATABASE.get_student_class_customizations(current_user()['username'])
+        customizations = DATABASE.get_student_class_customizations(
+            current_user()['username'], preview_class_as_teacher=session.get("preview_class"))
```

Comment:
Should we add it to https://github.com/hedyorg/hedy/blob/main/app.py#L1149 as well?

Reply:
Good point. Does this functionality make sense for the hour of code version? I'm not familiar with how that works.

Reply:
Perhaps not, since we don't really require a teacher for that indeed!